src/exasolvs/queryrenderer/SelectAppender.lua

```diff
@@ -90,12 +89,7 @@ function SelectAppender:_append_from(from)
 end
 
 function SelectAppender:_append_expression(expression)
-    local type = expression.type
-    if text.starts_with(type, "function_scalar") then
-        require("exasolvs.queryrenderer.ScalarFunctionAppender"):new(self._out_query):append_scalar_function(expression)
-    else
-        require("exasolvs.queryrenderer.ExpressionAppender"):new(self._out_query):append_expression(expression)
-    end
+    require("exasolvs.queryrenderer.ExpressionAppender"):new(self._out_query):append_expression(expression)
```

Comment:
Do you avoid assigning a local variable for `require("exasolvs.queryrenderer.ExpressionAppender")` by intention?
This may hide problems during amalgamation.